docs/docusaurus.config.ts

```diff
@@ -37,6 +37,14 @@ const config: Config = {
           sidebarPath: "./sidebars.ts",
           editUrl: "https://github.com/ChainSafe/lodestar/tree/unstable/docs/",
           routeBasePath: "/",
+          lastVersion: 'current',
+          versions: {
+            current: {
+              label: '1.7.0'
```

Comment:
shouldn't this match whats in our packages/cli/package.json?
What is the process for synchronizing this?